.travis.yml

```diff
@@ -29,8 +29,12 @@ install:
   - git config --global user.email "you@example.com"
   - git config --global user.name "Your Name"
   - sudo apt-get install -y tree
+  - sudo apt-get install -y xclip
```

Comment:
can you merge this with the other apt-get install so that it is faster?
``apt-get install -y tree xclip``